source/core/ut_utils.pks

```diff
@@ -381,6 +385,21 @@ create or replace package ut_utils authid definer is
   * Remove given ORA error from stack
   */
   function remove_error_from_stack(a_error_stack varchar2, a_ora_code number) return varchar2;
+  
+  /**
+  * Check if xml name is valid if not build a valid name
+  */
+  function get_valid_xml_name(a_name varchar2) return varchar2;
+  
+  /**
+  * Build valid XML column name as element names can contain letters, digits, hyphens, underscores, and periods
+  */
+  function build_valid_xml_name(a_preprocessed_name varchar2) return varchar2;
+  
+  /**
+  * Change string into unicode to match xmlgen format _00xx_
+  */
+  function char_to_xmlgen_unicode(a_character varchar2) return varchar2;  
```

Comment:
Do all 3 need to be public?
Best not to expose non-public code unless it's used outside.

Reply:
Only one